.github/workflows/ci.yml

```diff
@@ -0,0 +1,72 @@
+name: Scala CI
+
+on:
+  push:
+    branches: [ main ]
+    tags: ["v*"]
+  pull_request:
```

Comment:
Whoops is it a valid yaml? 😯 with no value for the key

Reply:
It most certainly is